app/javascript/redux/greetings/greetingsSlice.js

```diff
@@ -0,0 +1,24 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+const API_URL = 'http://localhost:3000/api/v1/greetings';
+
+const FETCH = 'hello-rails-react/greetings/FETCH';
+
+const initialState = {
+  greeting: '',
+};
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case `${FETCH}/fulfilled`:
+      return { ...state, greeting: action.payload };
+    default:
+      return state;
+  }
+};
+
+export const fetchGreeting = createAsyncThunk(FETCH, async () => {
```

Comment:
- [ ] Kindly adhere to the project requirements using the `redux/toolkit` but the `request` handling part of the `redux` is not correct, you should use the `createSlice` method provided by the `redux/toolkit` to manage the state properly and for this milestone, you can also take inspiration, or help from your one of the previous project you did in `React` module. I 💯 agree that, if the output is as expected as we want then what is the issue, I am sincerely telling you that as a code reviewer, we must ensure that you are following things properly because at this point of the `curriculum` Microverse aim is to ensure you solid learning skillset, working on the things over and over again helps the developer to strengthen their memory muscle and in your "Full Capstone Project" it will play a vital role in your project with team members to handle the flow of data efficiently and without prop-drilling.